main.js

```diff
@@ -0,0 +1,76 @@
+const bookForm = document.querySelector('#form');
+// const collection = document.querySelector('.table');
```

Comment:
- [ ] Please, kindly delete this old piece of commented code, to improve the cleanliness of the repo 💪